ngraph/frontend/onnx_import/src/op/constant.cpp

```diff
@@ -42,9 +42,14 @@ namespace ngraph
                         }
                         catch (const ngraph::ngraph_error& exc)
                         {
-                            NGRAPH_WARN << "Could not create an nGraph Constant for an ONNX "
-                                           "Constant node. Detailed error:\n"
-                                        << exc.what();
+                            NGRAPH_WARN
+                                << "\nCould not create an nGraph Constant for an ONNX Constant "
+                                   "node. "
```

Comment:
It could be better formatted (I know it's a fault of style-apply)

Reply:
Yes, it's a style-apply magic